build/common.jl

```diff
@@ -14,9 +15,18 @@ const TARBALL_REGEX = r"""
 const GH_RELEASE_ASSET_PATH_REGEX = r"""
     ^/(?<owner>[^/]+)/(?<repo_name>[^/]+)/
     releases/download/
-    (?<tag>[^/]+)$
+    (?<tag>[^/]+)/?$
     """x
 
+const REQUIRED_BASE_TRIPLETS = ("x86_64-linux-gnu", "aarch64-apple-darwin")
+const REQUIRED_JULIA_VERSIONS = (v"1.8", v"1.9")
+const REQUIRED_PLATFORMS = let
+    base_platforms = parse.(Platform, REQUIRED_BASE_TRIPLETS)
+    base_tags = [(; julia_version=string(v)) for v in REQUIRED_JULIA_VERSIONS]
+    [Platform(arch(p), platform_name(p); tags...)
+     for p in base_platforms, tags in base_tags][:]
+end
```

Comment:
Didn't spend much time refactoring this. If we can make the artifacts non-Julia specific this becomes much cleaner

Reply:
LGTM - it's not worth prettifying anyway